R/cancensus.R

```diff
@@ -102,6 +106,14 @@ cancensus.load <- function (dataset, level, regions, vectors=c(), geo=TRUE, form
   } else {
     result=dat
   }
+  if (length(vectors)>0) {
+   census_labels <- names(result)[grep("^v_", names(result))]
+   census_labels <- strsplit(census_labels, ": ")
+   census_labels <- as_data_frame(do.call(rbind, census_labels))
```

Comment:
Can't get bind_rows() to work for this, but `as_data_frame` here should namespace dplyr.